src/train/base_model.py

```diff
@@ -16,8 +14,8 @@ class ITrain(metaclass=ABCMeta):
     # and these kinds of punctuations
     # BTW： maybe use regular expression
     # TODO: keep maintaining for this object
-    FILTER_WORD = ['<root>', ',', '，', '.', '。', ';', '；', '!', '！', '?', '？',
-                   '\\', '、', '“', '”', '……']
+    # FILTER_WORD = ['<root>', ',', '，', '.', '。', ';', '；', '!', '！', '?', '？',
```

Comment:
Did you add different punctuation?

Reply:
NO!
I've replaced it with "string.punctuation" library and added some extra data cleaning functionality. Previously the data was not been cleaned by the function itself.